handbook/engineering/environments.md

```diff
@@ -0,0 +1,32 @@
+# Environment
+
+This page describes our different cloud environments.
+
+## Google Cloud
+
+We utilize multiple Google Cloud projects and folders to organize our workloads and manage access control.
+
+
+### Root Projects
+
+- **Sourcegraph**: Services for sourcegraph.com.
+- **sourcegraph-code-intel**: Services for Code Intel code execution.
+- **sourcegraph-managed-**: There are multiple `-managed` projects, one for each our managed deployments.
+- **TelligentSourcegraph**: Data pipeline and storage for pings and Cloud event logging.
+- **Sourcegraph Auxiliary**: Testing clusters and deployments.
+- **Universities**: Sourcegraph instances for universities. (Deprecated)
+- **sourcegraph-interviews**: Shared project for interviews
+
+### Folders
+Dynamic or temporary projects are created in the folders listed below.
+
+- **Engineers Projects**: Contains multiple engineers project. All projects must be prefixed with `$name-`.
```

Comment:
Do we have this folder today? Because I can't see it... although I also can't see most of projects listed here 😂

Reply:
I created it yesterday, I still have to sort out permissions, most projects/folders and even the org are not seen by members right now.

Reply:
If i'm understanding this right:

```suggestion
- **Engineers Projects**: Contains projects used by individual engineers. All projects must be prefixed with `$name-` (the name of the owner).
```